k8s/node.go

```diff
@@ -56,3 +59,13 @@ func (n *Node) GetURI() string {
 func (n *Node) GetK8sObject() *k8sapi.Avalanchego {
 	return n.k8sObj
 }
+
+// GetStakingURL
+func (n *Node) GetStakingURL() string {
+	return fmt.Sprintf("%s:%d", n.k8sObj.Status.NetworkMembersURI[0], constants.DefaultStakingPort)
+}
+
+// GetAPIURL
+func (n *Node) GetAPIURL() string {
+	return fmt.Sprintf("%s:%d", n.k8sObj.Status.NetworkMembersURI[0], constants.DefaultAPIPort)
```

Comment:
Can we just use `n.uri `instead of `n.k8sObj.Status.NetworkMembersURI[0]`?

Reply:
See above